### PR TITLE
Feature: Infoblox manage session with cookies in integration storage

### PR DESCRIPTION
**Pull request by jesusgpo**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
Now the infoblock integration is doing the authentication for each command execution, this has a very high impact on infoblox, that's why the cookies are stored in the context of the integration and loaded before each call, avoiding to login unless the cookies expire.

### CODE REVIEW

File: Packs/Infoblox/Integrations/Infoblox/Infoblox.py [hunk 1]
```diff
@@ -161,6 +164,69 @@ class IPv4AddressStatus(Enum):
...
+            and (context_cookies := integration_context.get("cookies"))
+        ):
+            load_cookies(client, context_cookies)
+
+        try:
+            response = func(client, *args, **kwargs)
+            save_cookies_to_context(client)
+            return response
+        except DemistoException as error:
+            if error.message is not None and error.message == INTEGRATION_AUTHORIZATION_EXCEPTION_MESSAGE:
```
Comment (MLainer1) [inline Packs/Infoblox/Integrations/Infoblox/Infoblox.py:223]:
```suggestion
            if error.message  and error.message == INTEGRATION_AUTHORIZATION_EXCEPTION_MESSAGE:
```

**Review by MLainer1 — CHANGES_REQUESTED** [review]
Thank you for your contribution.
Please attach a short demo video demonstrating your change.

**Comment by jesusgpo** [comment]
Hi may, yesterday I attached you the demo video by slack

**Comment by MLainer1** [comment]
Looks great! make sure to fix the pre-commit errors in the Github workflows.
Afterwards we can continue to merging the PR.

**Comment by MLainer1** [comment]
@jesusgpo Is it ready for merge from your side?

**Comment by jesusgpo** [comment]
Hi @MLainer1 

I have just updated the release, thank you for your help.

Kind regards

**Comment by Shellyber** [comment]
For the Reviewer: Trigger build request has been accepted for this contribution PR.

**Comment by Shellyber** [comment]
For the Reviewer: Successfully created a pipeline in GitLab with url: https://gitlab.xdr.pan.local/xdr/cortex-content/content/-/pipelines/1717709

**Comment by ShirleyDenkberg** [comment]
@jesusgpo Doc review completed.

**Comment by Shellyber** [comment]
For the Reviewer: Trigger build request has been accepted for this contribution PR.

**Comment by Shellyber** [comment]
For the Reviewer: Successfully created a pipeline in GitLab with url: https://gitlab.xdr.pan.local/xdr/cortex-content/content/-/pipelines/1789964

**Pull request merged**